tools/move-instance

```diff
@@ -972,6 +972,12 @@ def _CheckInstanceOptions(parser, options, instance_names):
 
     if options.nics:
       options.nics = cli.ParseNicOption(options.nics)
+      # allow user to override instance parameters by hardcoding the
+      # empty string as "ignore this setting" shortcut
+      for nic in options.nics:
+        for k, v in nic.items():
+          if not v:
```

Comment:
I would prefer to make it explicit that we are looking for an empty string here (`if not v == ""` or `if not len(v)`), because `not v` also matches things like `0` which can be legitimate values.

Reply:
oh yeah of course, fixed.

also, i think you meant `if v == ""` here, right?